corehq/apps/app_manager/static/app_manager/js/apps_base.js

```diff
@@ -19,4 +19,16 @@ $(function () {
     if (v2) {
         hqLayout.utils.setIsAppbuilderResizing(true);
     }
+
+    $(document).ajaxComplete(function(e, xhr, options) {
+        if (/edit_form_attr/.test(options.url) ||
```

Comment:
The url matching seems potentially fragile as we add new urls (or switch existing urls to use ajax). What would you think of doing something like firing an additional request to check the version number? It'd be another request, but a short one, and would be more future-proof.